dashboard/cron.py

```diff
@@ -61,8 +61,7 @@ def util_function(UDW_course_id, sql_string, mysql_table, table_identifier=None)
     # returns the row size of dataframe
     return f"inserted + {str(df.shape[0])} rows in table {mysql_table} for course {UDW_course_id}\n"
 
-# Exit with log if UDW dataset is missing courses
-def data_validation (incoming_courses,existing_courses):
+    # exit cron with log if UDW dataset is missing courses
```

Comment:
This is a fundamental error in coding:

No line of code will be executed after the return statement on line 62

Reply:
Nice catch!
That is not code I edited, but I should have seen that.
Thank you for informing me.

Reply:
In this line of code, I removed the unused cast of an Exception "as e", as that wasn't used.
It may be wise for future releases to include casting an Exception as e to widen the "net" for for caught exeptions.

https://stackoverflow.com/questions/18982610/difference-between-except-and-except-exception-as-e-in-python

Reply:
What do you mean by "That is not code I edited"? 

The removal of line 64 and line 65 is inside the 474e717d5acf3ae411ee1d16347d1f7b27c4476f commit.

Reply:
There is also a warning from Codacy for "Unreachable code".